contracts/Comptroller.sol

```diff
@@ -372,7 +372,7 @@ contract Comptroller is ComptrollerV5Storage, ComptrollerInterface, ComptrollerE
 
         // Keep the flywheel moving
         Exp memory borrowIndex = Exp({mantissa: CToken(cToken).borrowIndex()});
-        updateCompBorrowIndex(cToken, borrowIndex);
+        //updateCompBorrowIndex(cToken, borrowIndex);
```

Comment:
Please remove instead of comment, we have git for history :)

Reply:
Why not adding a flag to remove rewards on borrowing?

Reply:
I didn't want to mess with the storage layout since it's a common pitfall in proxy upgrades. The purpose of this upgrade is to move fast, potentially without test coverage. So it's better to do as little changes as possible.